### DISCUSSION TITLE
Add option to lint only changed LINES instead of files.

### DISCUSSION THREAD

**Discussion by ItamarShDev** [biomejs/biome#2120] [General]
We migrated to Biome and want to gradually fix the code. the easiest way would be to be able to fail the CI if the changed lines in the commit fail.

currently the --changed flag is working on the file itself

**Comment by ematipico** [comment]
We plan to add this feature, but it will be an additional feature, and it won't change how `--changed` works

**Comment by itse4elhaam** [comment]
Any plans on this feature?

it would be a killer.

**Comment by peterskogsberg** [comment]
any update? :)
